### PR TITLE
chore(deps): update `discord-api-types` and /voice

### PR DESCRIPTION
**Pull request by kyranet**
**Please describe the changes this PR makes and why it should be merged:**

This PR updates all packages' version of `discord-api-types` to `0.33.0`.

Also removes the `music-bot` example from `/voice`, since it uses the YouTube API.

**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

**Comment by ImRodry** [comment]
> Also removes the music-bot example from /voice, since it uses the YouTube API.

Couldn't you replace it with something like Spotify?

**Review by amishshah — APPROVED** [review]

**Comment by kyranet** [comment]
> Couldn't you replace it with something like Spotify?

My experience with Spotify is that it was impossible to make music bots that consume it. Could be wrong, but it could also be under the same legal issues as consuming YouTube's media, and it's not something we want to get into.

Furthermore, there are other examples which do not use ytdl.

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: packages/voice/examples/recorder/package.json [hunk 1]
```diff
@@ -15,8 +15,8 @@
 	"license": "MIT",
 	"dependencies": {
 		"@discordjs/opus": "^0.5.3",
-		"discord-api-types": "^0.22.0",
-		"discord.js": "^13.0.1",
+		"discord-api-types": "^0.30.0",
```
[the remark this answers is not in the document]
Comment (kyranet) [reply packages/voice/examples/recorder/package.json:18]: Depends on discord.js v13.7.0, which uses 0.30.0:

https://github.com/discordjs/discord.js/blob/e3cbd45e7de069432818148b9dedda03f630f9f7/package.json#L58

**Review by vladfrangu — APPROVED** [review]

**Comment by paulpv** [comment]
@kyranet You also need to remove the entry from the readme.md

Even though it did use the YouTube API, the example was useful and perhaps the code could have been kept and the youtube specific stuff just replaced with a generic "insert your player here" type of comment.

**Comment by kyranet** [comment]
> You also need to remove the entry from the readme.md

Want to open a PR for it, @paulpv?

> Even though it did use the YouTube API, the example was useful and perhaps the code could have been kept and the youtube specific stuff just replaced with a generic "insert your player here" type of comment.

We have an entire guide in which, [we practically spoonfeed the entire player thing](https://discordjs.guide/voice/audio-player.html#cheat-sheet), there's no need to keep identical code examples in two places.

**Pull request merged**